Makefile

```diff
@@ -0,0 +1,19 @@
+FE_NODE_ESLINT_DEPENDENCIES := @vue/cli-plugin-eslint @vue/eslint-config-typescript eslint-config-airbnb-base
+FE_NODE_STYLELINT_DEPENDENCIES := stylelint-config-standard
+
+hint:
+	( \
+		echo "Actions:"; \
+		echo "  make fe – link front end configuration files to the root of the project"; \
+	)
+
+.PHONY: fe
+fe:
+	( \
+		npm install --prefix ../ --save-dev $(FE_NODE_ESLINT_DEPENDENCIES) $(FE_NODE_STYLELINT_DEPENDENCIES); \
+		echo "npm dependencies successfully installed"; \
+		ln -s fe/.stylelintrc ../.stylelintrc; \
+		ln -s fe/.eslintrc.js ../.eslintrc.js; \
+		ln -s fe/.editorconfig ../.editorconfig; \
```

Comment:
Copying is not an option for our case. This repository will be used as submodule, thus the link will be much more convenient as it will be updated once the latest changes are pulled.

Reply:
Also, I'll replace `ln -s` with `ln` as the ESLint is unable to work with symbolic links.